versioncontrol/peer.go

```diff
@@ -55,6 +56,13 @@ type Version struct {
 	URL     string `user:"true" help:"URL for specific binary" default:""`
 }
 
+// Rollout represents the state of a version rollout.
+type Rollout struct {
+	Seed   string  `user:"true" help:"random 32 byte, base64-encoded string"`
+	Cursor int     `user:"true" help:"percentage of nodes which should roll-out to the target version"`
```

Comment:
would this Cursor also need to be changed to string?

Reply:
Yes, this will have to change as well

Reply:
Okay, now I'm thinking maybe not. We discussed having the user-facing value be a percentage instead of the mask for human-readability. We also discussed a script that would generate the k8s env var section so that one could copy/paste it, so it would make sense that that script takes a percentage as input; the question is then, should the config value be a percentage as well? I think it would be beneficial to persist the human readability through to the k8s env vars just so that it's obvious from a glance at something that's under version-control what's going on.